src/context/layoutContext.tsx

```diff
@@ -0,0 +1,38 @@
+import { createContext, useContext } from 'react';
+import { CartesianViewBox } from '../util/types';
+import { XAxisProps, YAxisProps } from '../index';
+
+export type XAxisMap = {
+  [axisId: string]: XAxisProps;
+};
+
+export type YAxisMap = {
+  [axisId: string]: YAxisProps;
+};
```

Comment:
Are these types specific to the layoutContext? I expect them to be used in many other places - should we put them into a more generic location to increase reusability / visibility?

Reply:
Done